apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../../components';
+import EventBullet from '../../../components/event-bullet/event-bullet';
+import './event-message-list.scss';
+
+/* eslint-disable-next-line */
+export interface EventMessageListProps {}
+
+/*TODO: Remove this and replace with query of event message channels*/
+
+const messageChannels = [
+  {
+    eventTitle: 'Clean up of Rogers Park',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 1, 8, 20, 30, 1),
+  },
+  {
+    eventTitle: 'Lets do this guys!',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 5, 8, 20, 30, 1),
+  },
+  {
+    eventTitle: 'My god please clean this...',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 7, 8, 20, 30, 0),
+  },
+];
+
+export const EventMessageList = (props: EventMessageListProps) => {
+  const [searchString, setSearchString] = useState<string>('');
+
+  return (
+    <div className="event-messages-container">
+      <ChatHeader eventTitle="Event Messages" />
+      <div className="page-container">
+        <CustomTextInput
+          title=""
+          fieldName="searchField"
+          value={searchString}
+          hasErrors={null}
+          errorMessage={null}
+          onChange={(e) => setSearchString(e.target.value)}
+          placeholder="Search"
+        />
+        {messageChannels.map((messageChannel) => {
+          if (messageChannel.eventTitle.toLowerCase().includes(searchString.toLowerCase()))
```

Comment:
Do you need to have a case for the empty string? Check @leojrsilao's file [Here](https://github.com/RGPosadas/Mull/blob/43a1593990209fee64f56b4b6593735cc03e9d4d/apps/mull-ui/src/app/pages/direct-message/direct-message.tsx)

Reply:
Also, we could probably add some no results text if it doesn't find anything.

Reply:
Empty string will just display all the events channels that it has available (static for now)